README.md

```diff
@@ -200,12 +204,20 @@ For a full list of contributors refer to the [authors page](./AUTHORS.md)
 
 ## Issues
 
+### Usage Questions
+
+Pose questions and help answer them on [Stack
+Overflow](https://stackoverflow.com/questions/tagged/fairlearn) with the tag
+`fairlearn` or on [Gitter](https://gitter.im/fairlearn/community#).
+
 ### Regular (non-security) issues
 
-Please submit a report through [GitHub issues](https://github.com/fairlearn/fairlearn/issues). A maintainer will respond promptly as follows:
+Issues are meant for bugs & feature requests. Please submit a report through
```

Comment:
We should probably update the language here to match the current experience with submitting issues. For example, I'm not completely sure whether there's still the "triage" step. Also, it is a bit strange to only have two bullets here given that we have 6 different template categories. It might be better to actually list all the 6 template categories (and say that security and question are treated differently)?

Reply:
We actually do respond on every issue. Perhaps the responses aren't quite sufficient in terms of providing expectations. I'll update with the relevant issue categories (excluding the blank template)

Reply:
We actually try our best at responding to all issues and triaging, but it's not an exact science due to the many different things on our plates. I've taken that out and replaced it with a simpler paragraph. LMK what you think!